src/game/game.cpp

```diff
@@ -935,10 +935,12 @@ namespace game
                     int millis = lastmillis-d->lastres[WR_BURN];
                     size_t seed = size_t(d) + (millis/50);
                     float pc = 1, amt = (millis%50)/50.0f, intensity = 0.75f+(detrnd(seed, 25)*(1-amt) + detrnd(seed + 1, 25)*amt)/100.f;
-                    if(burntime-millis < burndelay) pc *= float(burntime-millis)/float(burndelay);
+                    // When playing on servers with an older version, burndelay can be 0, thus add a check to prevent division by 0
+                    int _burndelay = std::max(burndelay, 1);
```

Comment:
The `_` prefix is preserved for standard-specified things (as per the ISO C standard) IIRC. Please rename.

Reply:
to what?

Reply:
ok, changed them to `fixed_burndelay`